force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,10 +58,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
+        // Initialize largestNum as null 
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 >= num2){
+            largestNum = num1;
+        }
+        if (num1 < num2) {
+            largestNum = num2;
+        }
+        return largestNum;
```

Comment:
Challenge for you. Take a look at the Math.max method. Can you do this with one line.